### PR TITLE
Add clarity to cipher rejection message

### PR DESCRIPTION
**Pull request by stu-gott**
**What this PR does / why we need it**:

Add clarity to error message when rejecting a cipher based on TLS settings.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2242207

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go [hunk 1]
```diff
@@ -117,12 +117,12 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
...
 			Entry("with unspecified minTLSVersion but non empty ciphers",
 				&v1.TLSConfiguration{Ciphers: []string{tls.CipherSuiteName(tls.TLS_AES_256_GCM_SHA384)}},
-				"You cannot specify ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
+				"You cannot specify non-compliant ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
 				-1,
 			),
 			Entry("with specified ciphers and minTLSVersion = 1.3",
 				&v1.TLSConfiguration{Ciphers: []string{tls.CipherSuiteName(tls.TLS_AES_256_GCM_SHA384)}, MinTLSVersion: v1.VersionTLS13},
-				"You cannot specify ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
+				"You cannot specify non-compliant ciphers when spec.configuration.tlsConfiguration.minTLSVersion is empty or VersionTLS13",
```
Comment (geetikakay) [inline pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go:125]: sorry for jumping in between , is there a way to identify and print non compliant cipher.

**Comment by stu-gott** [comment]
Re-reading the code, I am now of the opinion that the original error message is correct. The code is checking if any cipher at all was defined in the manifest, and throwing an error if one is present. thus "you cannot specify ciphers..." is correct in the first place.

File: pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go (hunk 1 above)
Comment (stu-gott) [reply pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go:125]: Thanks for the question. digging to check the name made it clear this PR is not needed.

**Comment by 0xFelix** [comment]
See https://github.com/kubevirt/hyperconverged-cluster-operator/issues/2553

**Pull request closed without merging**